engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -6654,9 +6758,9 @@ void demon_hunter_t::target_mitigation( school_e school, result_amount_type dt,
       s->result_amount *= 1.0 + spec.fiery_brand_dr->effectN( 1 ).percent();
     }
 
-    if ( td->debuffs.void_reaver )
+    if ( td->debuffs.frailty->up() && talent.vengeance.void_reaver )
```

Comment:
This should probably be check() rather than up() due to not wanting to have another source of benefit tracking. (Think it uses up() in the damage reduction calculations.)